jsonschema-hyperschema.xml

```diff
@@ -863,69 +924,54 @@ GET /foo/
                 </section>
             </section>
 
-            <section title="Submission Form Properties">
+            <section title="Data Submission Properties">
                 <t>
-                    The following properties also apply to Link Description Objects, and provide functionality analogous to <xref target="W3C.CR-html5-20140731">HTML forms</xref>, by providing a means for making a request with client- or user-selected information.
+                    The following properties also apply to Link Description Objects,
+                    and provide the ability to describe data for submission to the
+                    target resource for processing, independent of manipulating the
+                    target resource's representation.
                 </t>
 
-                <section title="method" anchor="method">
+                <section title="form" anchor="form">
+                    <t>
+                        The value of this keyword is a boolean.
+                    </t>
                     <t>
-                        This property specifies that the client can construct a templated query or non-idempotent request to a resource.
+                        Setting this keyword to true indicates that this link has
+                        semantics analogous to
+                        <xref target="W3C.REC-html5-20141028">HTML POST forms</xref>.
+                        This imposes additional constraints that are not present if
```

Comment:
I'm afraid I just don't understand your distinction.

It is already the case that `submissionSchema` is the only way to accomplish the goal of sending data to an executable resource.  That is its purpose, and the presence or value of `form` is completely irrelevant to the semantics of the submission keywords.

The only thing that `"form": true` changes is forbidding any other operations.  If that's not a constraint I don't know what is.